test/unit/browser/cache/caching_strategy_common_test.dart

```diff
@@ -1,3 +1,18 @@
+// Copyright 2018 Workiva Inc.
```

Comment:
Technically, I think these are supposed to be the years the file was modified in, so if it was created in 2016 and no modified until now it would be `2016, 2018` or if it was also modified in 2017 it would be `2016-2018`.